hawkbit-dmf-amqp/src/main/java/org/eclipse/hawkbit/amqp/AmqpConfiguration.java

```diff
@@ -58,6 +60,9 @@
     @Autowired
     private ConnectionFactory rabbitConnectionFactory;
 
+    @Autowired
+    private ErrorHandler errorHandler;
```

Comment:
Calling the private method won't work yes, because there is no spring-proxy around which could check if there is already a bean `errorHandler` 

One way could also be setting the parameter to the method 
```
@Bean(name = { "listenerContainerFactory" })
    public RabbitListenerContainerFactory<SimpleMessageListenerContainer> listenerContainerFactory(
            final ErrorHandler errorHandler) {
        return new ConfigurableRabbitListenerContainerFactory(amqpProperties, rabbitConnectionFactory, errorHandler);
    }
```

but in general it looks weird having the default `errorHandler` within the AmqpConfiguration, what about moving the 
```
    @Bean
    @ConditionalOnMissingBean
    public ErrorHandler errorHandler() {
        return new ConditionalRejectingErrorHandler();
    }

```
to the autoconfiguration or just handle `null` because `null` is already handled in the `AbstractRabbitListenerContainerFactory`

```
@Override
	public C createListenerContainer(RabbitListenerEndpoint endpoint) {
		C instance = createContainerInstance();

		if (this.connectionFactory != null) {
			instance.setConnectionFactory(this.connectionFactory);
		}
		if (this.errorHandler != null) {
			instance.setErrorHandler(this.errorHandler);
		}
```

And if the `errorHandler` is not set then the default is used 
```
private ErrorHandler errorHandler = new ConditionalRejectingErrorHandler();
```